packages/client/mutations/AutogroupMutation.ts

```diff
@@ -7,6 +7,9 @@ graphql`
   fragment AutogroupMutation_meeting on AutogroupSuccess {
     meeting {
       id
+      resetReflectionGroups {
```

Comment:
+1 As far as I can see it, the client needs this only to know it can reset the autogroups, so it could just be a boolean?

Reply:
It could be, but we'd have to add an additional field to the `RetrospectiveMeeting` object. We're using `resetReflectionGroups` to reset the reflections to their original groups, so I thought it'd be best to check whether it exists rather than add an additional field. 

I'm open to using a boolean though if you feel this would be inefficient on the client.